dll/win32/shlwapi/ordinal.c

```diff
@@ -5322,6 +5322,25 @@ HRESULT WINAPI SHPropertyBag_ReadLONG(IPropertyBag *ppb, LPCWSTR pszPropName, LP
 }
 
 #ifdef __REACTOS__
+/**************************************************************************
+ *  SHPropertyBag_Delete (SHLWAPI.535)
+ */
+HRESULT WINAPI SHPropertyBag_Delete(IPropertyBag *ppb, LPCWSTR pszPropName)
+{
+    VARIANT vari;
+
+    TRACE("%p %s\n", ppb, debugstr_w(pszPropName));
+
+    if (!ppb || !pszPropName)
+    {
+        ERR("%p %s\n", ppb, debugstr_w(pszPropName));
+        return E_INVALIDARG;
+    }
+
+    V_VT(&vari) = VT_EMPTY;
+    return IPropertyBag_Write(ppb, pszPropName, &vari);
```

Comment:
Do you have testcases for this?

Reply:
No, I haven't.
I will add it in the future.

Reply:
`modules/rostests/winetests/shlwapi/ordinal.c`

This will work in the future.

Reply:
Added `SHPropertyBag` testcase.